cscs-checks/apps/vasp/vasp_check.py

```diff
@@ -8,16 +8,19 @@
 
 
 class VASPCheck(rfm.RunOnlyRegressionTest):
-    keep_files = ['OUTCAR']
-    maintainers = ['LM']
-    tags = {'scs'}
-    strict_check = False
+    scale = parameter(['small', 'large'])
+    variant = parameter(['maint', 'prod'])
```

Comment:
Do we need to have `maint` and `prod` variants? I have removed it from other tests, because we have not been practically updating the references for the `maint` variants.

Reply:
I have removed the two variants and kept `tags = {'maintenance', 'production'}`